### PR TITLE
MHV-55434 the txt file download revision for blue button

### PR DESCRIPTION
**Pull request by Wyzardsleeves**
## Summary

- Updated content/data for consolidated reports.
- Is not a bug.
- Updated existing content and added new lines of code.
- MHV Medical Records.

### [MHV-MHV-55434](https://jira.devops.va.gov/browse/MHV-55434) - Back Button focus issue all domains ###

Screenshot 2024-03-28 at 3 28 16 PM

Screenshot 2024-03-28 at 3 28 48 PM

## Testing done

-  Manual testing

## What areas of the site does it impact?
MHV Medical Records

### CODE REVIEW

File: src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js [hunk 1]
```diff
@@ -137,7 +168,7 @@ ${record.notes.map(note => `${note}`)}
   const parseAllergies = records => {
     return `
 ${txtLine}
-Allergies
+4) Allergies
 
 If you have allergies that are missing from this list, send a secure message to your care team.
 ${records.map(
```
Comment (mmoyer-va) [inline src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js:174]: All of these `records.map` calls are being joined with commas inside the string literals. You can put `.join('')` at the end to get rid of them (see Allergies.jsx) for an examples of this.
image

**Review by mmoyer-va — CHANGES_REQUESTED** [review]

File: src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js (hunk 1 above)
Comment (Wyzardsleeves) [reply src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js]: Thanks for pointing out the oversight.

**Comment by Wyzardsleeves** [comment]
Screenshot 2024-04-04 at 12 19 59 PM

File: src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js [hunk 2]
```diff
@@ -117,56 +154,61 @@ Summary
...
-${records.map(
-      record => `
+${records
+      .map(
+        record => `
 ${record.name}
 ${txtLineDotted}
 Date entered: ${record.date}
-Signs and symptoms: ${record.reaction.map(reaction => `${reaction}`)}
+Signs and symptoms: ${record.reaction.map(reaction => `${reaction}`).join('')}
```
Comment (mmoyer-va) [inline src/applications/mhv/medical-records/util/txtHelpers/downloadRecords.js:191]: Don't use `.join('')` on this one as it is a real list. Instead use `.join(', ')`
image

**Review by mmoyer-va — CHANGES_REQUESTED** [review]

**Review by mmoyer-va — DISMISSED** [review]
Looks good!

**Review by tynyttie — APPROVED** [review]
Looks good.

**Pull request merged**